icu4c/source/i18n/unicode/ucal.h

```diff
@@ -1617,6 +1617,100 @@ U_CAPI int32_t U_EXPORT2
 ucal_getTimeZoneIDForWindowsID(const UChar* winid, int32_t len, const char* region,
                                 UChar* id, int32_t idCapacity, UErrorCode* status);
 
+#ifndef U_HIDE_DRAFT_API
+/**
+ * The option used by ucal_getTimeZoneOffsetFromLocal to specify
+ * how to interpret an input time when it does not exist or ambiguous
+ * around a time zone transition.
```

Comment:
```suggestion
 * Options used by ucal_getTimeZoneOffsetFromLocal and BasicTimeZone::getOffsetFromLocal()
 * to specify how to interpret an input time when it does not exist, or when it is ambiguous,
 * around a time zone transition.
```

Reply:
done

Reply:
Wait -- none of the things in this file that you noted as "done" seem actually done.